### PR TITLE
packages: Remove git hash from version

### PR DESCRIPTION
**Pull request by jcvenegas**
Do not add  the `commit id` to the package version, this help to install an specific kata version when
using repositories that provide different  versions like packageclould.

- Define runtime deps with the exact set of versions
of a release.

- remove obs release counter,  we already use our counter for rpms and debs and dnf  gets confused
if we used. 

Fixes: #453

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Comment by jcvenegas** [comment]
\test

**Comment by grahamwhaley** [comment]
l g t m
But, I'd like the more packaging aware folks to ack that this is a move in the right direction.

File: obs-packaging/scripts/pkglib.sh [hunk 1]
```diff
@@ -291,18 +291,23 @@ function pkg_version() {
...
 
 	pkg_version="${project_version}"
 
 	if [ -n "$commit_id" ]; then
-		pkg_version+="+git.${commit_id:0:7}"
+		# see https://github.com/openSUSE/obs-service-tar_scm/blob/master/tar_scm.service.in
+		# %h format
+		#obs short commit length is:
+		obs_short_commit_length=10
+		pkg_version+="+git.${commit_id:0:${obs_short_commit_length}}"
```
Comment (marcov) [inline obs-packaging/scripts/pkglib.sh]: I am not sure to understand this...why did you change from 7 to 10? `%h` is still 7 characters long.
